risc0/cargo-risczero/README.md

```diff
@@ -64,3 +64,18 @@ Use the `build-toolchain` command to build the toolchain locally for your host.
 [risc-zero]: https://risczero.com
 [install-rust]: https://doc.rust-lang.org/cargo/getting-started/installation.html
 [rust-starter]: https://github.com/risc0/risc0/tree/main/templates/rust-starter
+
+## build
+
+Use the `build` command to build a guest code for the target `riscv32im-risc0-zkvm-elf`. The resulting ELF binary will be saved on `elfs/`. Warning: this requires having docker installed.
```

Comment:
I think the other requirement is that the have run `cargo risczero install` first right?

Reply:
Isn't the toolchain only required in the docker image?

Reply:
ahhh yes thats correct.